### PR TITLE
Tweak error message for missing fallback

### PR DESCRIPTION
**Pull request by sophiebits**

### CODE REVIEW

**Comment by sophiebits** [comment]
I should try to include the component stack.

**Comment by acdlite** [comment]
Maybe we can get rid of the passive voice while we're at it.

Alt suggestion:

> ComponentName suspended while rendering, but React doesn't know what to render in its place. Add \<Suspense fallback=... /\> somewhere higher in the tree.

**Comment by sophiebits** [comment]
How's this.

File: packages/react-reconciler/src/ReactFiberUnwindWork.js [hunk 1]
```diff
@@ -315,8 +316,14 @@ function throwException(
       workInProgress = workInProgress.return;
     } while (workInProgress !== null);
     // No boundary was found. Fallthrough to error mode.
+    // TODO: Use invariant so the message is stripped in prod?
```
[the remark this answers is not in the document]
Comment (sophiebits) [reply packages/react-reconciler/src/ReactFiberUnwindWork.js]: seemsreasonable

**Pull request merged**